tests/tests.py

```diff
@@ -2245,12 +2254,90 @@ def test_utils(self):
                          'Reserved')
         self.assertEqual(ivre.utils.get_addr_type('255.255.255.255'),
                          'Broadcast')
+        # ipv6
+        self.assertEqual(ivre.utils.get_addr_type('::'),
+                         'Unspecified')
+        self.assertEqual(ivre.utils.get_addr_type('::1'), 'Loopback')
+        self.assertIsNone(ivre.utils.get_addr_type('::ffff:8.8.8.8'))
+        self.assertEqual(ivre.utils.get_addr_type('64:ff9b::8.8.8.8'),
+                         'Well-known prefix')
+        self.assertEqual(ivre.utils.get_addr_type('100::'),
+                         'Discard (RTBH)')
+        self.assertEqual(ivre.utils.get_addr_type('2001::'),
+                         'Protocol assignements')
+        self.assertIsNone(ivre.utils.get_addr_type('2001:4860:4860::8888'))
+        self.assertEqual(ivre.utils.get_addr_type('2001:db8::db2'),
+                         'Documentation')
+        self.assertEqual(ivre.utils.get_addr_type('fc00::'),
+                         'Unique Local Unicast')
+        self.assertEqual(ivre.utils.get_addr_type('fe80::'),
+                         'Link Local Unicast')
+        self.assertEqual(ivre.utils.get_addr_type('ff00::'),
+                         'Multicast')
 
         # ip2int() / int2ip()
         self.assertEqual(ivre.utils.ip2int("1.0.0.1"), (1 << 24) + 1)
         self.assertEqual(ivre.utils.int2ip((1 << 24) + 1), "1.0.0.1")
         self.assertEqual(ivre.utils.ip2int('::2:0:0:0:2'), (2 << 64) + 2)
         self.assertEqual(ivre.utils.int2ip((2 << 64) + 2), '::2:0:0:0:2')
+        self.assertEqual(
+            ivre.utils.int2ip6(0x1234567890ABCDEFFED0000000004321),
+            '1234:5678:90ab:cdef:fed0::4321'
+        )
+        # ip2bin
+        # unicode error
+        self.assertEqual(
+            ivre.utils.ip2bin(b'\x33\xe6\x34\x35'),
+            b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\xff\xff3\xe645'
+        )
+        with self.assertRaises(ValueError):
+            ivre.utils.ip2bin(b'\xe6')
+        # else case
+        with self.assertRaises(ValueError):
+            ivre.utils.ip2bin(b'23T')
+        self.assertEqual(
+            ivre.utils.ip2bin(b'23TT'),
+            b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\xff\xff23TT'
+        )
+        self.assertEqual(ivre.utils.ip2bin(b'T3STTESTTESTTEST'),
+                         'T3STTESTTESTTEST')
```

Comment:
```suggestion
                         b'T3STTESTTESTTEST')
```
(since the output is supposed to be "binary".